### ISSUE TITLE
Create a One Sheet: Engage

### ISSUE THREAD

**Issue by MSS915** [hackforla/product-management#65]
### Dependency
Waiting on logo https://github.com/hackla-engage/start-here/issues/11

### Overview
Create the project one sheet so it can be shared with stakeholders and funders.

### Action Items
- [x] Check to confirm project does not already have a one sheet
- [x] Speak to the team to get an overview of the project
- [x] Include project logo
- [x] Create history/last 6 months
- [x] Create roadmap/next 6 months
- [x] Include last updated date

### Resources/Instructions
Check issue #3 for sample one sheets

[original draft Engage One Sheet](https://docs.google.com/document/d/18ICYthZPKQgtJnhV1PpMGUqIXeN7hAgRNC0E4JsIids/edit)
[Engage project board issue](https://github.com/hackla-engage/start-here/issues/2)
[NEW draft Engage One Sheet](https://docs.google.com/document/d/1Z8TZKnh4n9LNTe6WaEqb4rKI8cb3pMIEnVtabqSrMk8/edit?usp=sharing)
[Issue for creating new logo](https://github.com/hackla-engage/start-here/issues/11)

**Comment by ktjnyc** [comment]
We are handling this through the [Engage project board issue](https://github.com/hackla-engage/start-here/issues/2)

**Comment by ktjnyc** [comment]
Updates made to NEW draft Engage One Sheet

**Comment by ExperimentsInHonesty** [comment]
fyi, while closing my old tabs I found this doc: https://docs.google.com/document/d/1NOu1xOQuOllZvrIpybfK0eY37TWqKlmWc056UYpOHtA/edit 
with this (dated end of july 2020

> Engage is a platform to help connect residents (and other stakeholders) to local government. Instead of requiring constituents to be present at meetings, which presents a host of challenges for many groups of people, why not also allow them to voice their opinions in electronic form? Most local governments lack the resources and technical skills to create a platform that acknowledges peoples' interests and allows them to reply to current issues. Engage also makes reading the comments from stakeholders more efficient, by grouping comments about the same issues with the same sentiment about the issue.

**Comment by ktjnyc** [comment]
Progress: Updated based upon notes from discussion with Bonnie.
Blocks:

- Need to discuss on team meeting updates and confirm pilot activities
- Need to discuss project logo
- Need to discuss use of engage@engage.town for contact email

Availability: pretty open this week
ETA: Can make next round of updates by Friday afternoon.

**Comment by ktjnyc** [comment]
Changes made during team meeting tonight.

**Comment by ExperimentsInHonesty** [comment]
@ktjnyc I updated the color to match the header color on the website.  I also linked the url.  Seems like we are waiting for the logo.  So 
1. please make a new issue for creating a logo on the [engage repo](https://github.com/hackla-engage/start-here) (I thought Bryon had done that, but I can't find it).
2. add a link to the logo issue to the top of this issue as a dependency and add an task item to release this issue from the icebox on the logo issue, when its finished.

**Comment by ktjnyc** [comment]
Issue for new logo created and tasks added to update the one sheet and this issue status when complete.

**Comment by ExperimentsInHonesty** [comment]
@ktjnyc I moved this to the icebox and added the dependency to the top of the issue.  You can move it back after thats released and resolved.

**Comment by ktjnyc** [comment]
@ExperimentsInHonesty Ryan added the logo to the One Sheet so the dependency is lifted and ready for your final review.

**Comment by ExperimentsInHonesty** [comment]
@ktjnyc Words are probably fine.  Please take a look at how some of the other one sheets are formatted, and take another pass at the formatting.

**Comment by ktjnyc** [comment]
Progress: Was able to make some updates
Blocks: Availability
Availability: 
ETA: Updates can be made by Friday 2/26

**Comment by ktjnyc** [comment]
Changes made:
- Color matched the headers and bolded font to the color of the project name in the logo. 
- Added the divider line between the overview and history/roadmap sections
- Moved the project logo down

**Comment by ktjnyc** [comment]
Updated the links and last edited date.  
File PDF'd and added to [repo](https://github.com/hackforla/product-management/tree/master/project-one-sheets) 
Updated [one sheet tracker file](https://docs.google.com/spreadsheets/d/1OJOKJDhUzzMzaFwmbGoOdh5Ohh53LCwS4u7LulVvBsQ/edit#gid=0) with status and approval dates

**Comment by ExperimentsInHonesty** [comment]
@ktjnyc thanks!
